src/js/components/Button/index.d.ts

```diff
@@ -64,7 +64,7 @@ export interface ButtonProps {
   primary?: boolean;
   reverse?: boolean;
   secondary?: boolean;
-  size?: 'small' | 'medium' | 'large';
+  size?: 'small' | 'medium' | 'large' | 'xlarge';
```

Comment:
If we are saying that Button supports 'xlarge' as a valid `size`, should base.js have a corresponding button xlarge defined? 

Or should we remove 'xlarge' and instead accept a string and document that string as a custom property defined in `theme.button.size`?
```suggestion
  size?: 'small' | 'medium' | 'large' | string;
```

Reply:
I like the string option